### PR TITLE
Fix: Extended Support for Layer Import for  TF and keras (#293)

### PR DESCRIPTION
**Pull request by Ram81**
@utsavgarg please review.

### CODE REVIEW

File: caffe_app/views/import_prototxt.py [hunk 1]
```diff
@@ -469,6 +469,16 @@ def ContrastiveLoss(layer):
     return params
 
 
+def Concat(layer):
+    params = {}
+    if (layer.concat_param.axis is not None):
+        params['axis'] = layer.concat_param.axis
+    else:
+        # default value for axis of dropout in caffe
```
Comment (utsavgarg) [inline caffe_app/views/import_prototxt.py]: How is this related to dropout?

Comment (Ram81) [reply caffe_app/views/import_prototxt.py]: Sorry I accidentally wrote dropout there it should be concat
